specifyweb/stored_queries/query_construct.py

```diff
@@ -17,7 +17,12 @@ def _safe_filter(query):
         return query.first()
     raise Exception(f"Got more than one matching: {list(query)}")
 
-class QueryConstruct(namedtuple('QueryConstruct', 'collection objectformatter query join_cache tree_rank_count internal_filters')):
+def get_treedef(collection, tree_name):
+    return (collection.discipline.division.institution.storagetreedef
+            if tree_name == 'Storage' else
+            getattr(collection.discipline, tree_name.lower() + "treedef"))
```

Comment:
Just curious– what is the purpose of this check? 

`storagetreedef` is the same even if you construct it with 
```py
getattr(collection.discipline, tree_name.lower() + "treedef"))
```